aws-sources-operator/bundle/0.1.0/aws-sources-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -262,8 +262,8 @@ spec:
   - aws
   - triggermesh
   links:
-  - name: Aws Event Sources Operator
-    url: https://aws-event-sources-operator.domain
+  - name: Aws Sources Operator
```

Comment:
Capitalize AWS

Reply:
let me check if there is some linting issue here.

Reply:
this line was automatically converted to title case by the tooling.

Reply:
:trollface:
As long as no one complains about "AWS" being cased like that.